client-app/core/constants/notifications.ts

```diff
@@ -1 +1,8 @@
 export const DEFAULT_NOTIFICATION_DURATION = 15000;
+
+export const UNREAD_COUNT_CACHE_ID = 'pushMessages:{"unreadOnly":true}';
+export const UNREAD_COUNT_WITH_HIDDEN_CACHE_ID = 'pushMessages:{"withHidden":true,"unreadOnly":true}';
+
+export const ROOT_QUERY_CACHE_ID = "ROOT_QUERY";
```

Comment:
Is this ROOT_QUERY relates to all query or to push messages only?

Reply:
it relates to all queries, moved it to `@/core/api/graphql/consts`